keyboards/exclusive/e6_rgb/rules.mk

```diff
@@ -11,29 +11,24 @@ MCU = atmega32u4
 #   ATmega328P   USBasp
 BOOTLOADER = atmel-dfu
 
-# Do not put the microcontroller into power saving mode
-# when we get USB suspend event. We want it to keep updating
-# backlight effects.
-#OPT_DEFS += -DNO_SUSPEND_POWER_DOWN
-
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration(+1000)
-MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
-EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
-CONSOLE_ENABLE = no         # Console for debug(+400)
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
 SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-NKRO_ENABLE = yes            # USB Nkey Rollover
+NKRO_ENABLE = no 		    # USB Nkey Rollover
```

Comment:
Why was this disabled?

Reply:
someone reported their boards can't work under boot mode. so it was turned off for their requests.

Reply:
Outright disabling NKRO is not the right solution here, I feel. I would prefer to find out why that is happening instead. Possibly they just need to toggle NKRO off in order for the board to work in the BIOS (I assume that's what you mean by "boot mode").

Reply:
yes, they can't use the keyboard while entering the BIOS. And they did not know how to build the firmware (they are heavily relying on the qmk configurator), thus I have to turn it OFF.

Reply:
Well, the NKRO toggle keycode available in the Configurator may help with this.

Reply:
thanks, I did not aware the existence of such keycode. The NKRO was turned to YES.